File: MODUL 6/Tugas Modul 6 Kelompok 49/Java/Tugas_Kelompok49/src/tugas_kelompok49/Tugas_Kelompok49.java
package tugas_kelompok49;
import java.util.Scanner;

public class Tugas_Kelompok49 {

    public static void main(String[] args) {
        Mahasiswa call = new Mahasiswa();
        Scanner in = new Scanner (System.in);
        System.out.println("----------------Input Data----------------");
        System.out.print("Nama : ");
        String a = in.nextLine();
        System.out.print("NIM : ");
        String b = in.nextLine();
        System.out.print("Umur(Tahun) : ");
        String c = in.nextLine();
        System.out.print("Hobi : ");
        String d = in.nextLine();
        System.out.print("Asal : ");
        String e = in.nextLine();
        System.out.println("------------------------------------------");
        call.seta(a);
        call.setb(b);
        call.setc(c);
        call.setd(d);
        call.sete(e);
        
        String f = call.geta();
        String g = call.getb();
        String h = call.getc();
        String i = call.getd();
        String j = call.gete();
        
        System.out.println("");
        System.out.println("");
        System.out.println("----------------Data Mahasiswa----------------");
        System.out.println("Nama : "+f);
        System.out.println("NIM : "+g);
        System.out.println("Umur : "+h+" Tahun");
        System.out.println("Hobi : "+i);
        System.out.println("Asal : "+j);
        System.out.println("------------------------------------------");
        System.out.println("Mahasiswa dengan nama "+f+" Yang memiliki NIM "+g+", serta berumur "+h+" Tahun dan berasal dari "+j+" Memiliki Hobi "+i);
        
        
        

        
    }
    
}

File: MODUL 6/Tugas Modul 6 Kelompok 49/Python/Tugas_Kelompok49/Tugas_Kelompok49/Tugas_Kelompok49.py
class Mahasiswa:
    def __init__ (self): 
        self.a = ""
        self.b = ""
        self.c = ""
        self.d = ""
        self.e = ""
        self.f = ""


    def geta (self):         
        return self.a
    def seta (self,a):      
        self.a = a

    def getb (self):
        return self.b
    def setb (self,b):
        self.b = b

    def getc (self):
        return self.c
    def setc (self,c):
        self.c = c

    def getd (self):         
        return self.d
    def setd (self,d):      
        self.d = d

    def gete (self):
        return self.e
    def sete (self,e):
        self.e = e

    def getf (self):
        return self.f
    def setf (self,f):
        self.f = f

    def getg (self):
        return self.g
    def setg (self,g):
        self.g = g



#Disini Dimulai Source Code utamanya
c = input("Masukkan Nomor Kelompok anda:")
print('-' * 40)
print("Masukkan Data anggota pertama:")
a = input("Nama\t\t:")
b = input("NIM\t\t:")
f = input("Asal\t\t:")
print("Masukkan Data anggota kedua:")
d = input("Nama\t\t:")
e = input("NIM\t\t:")
g = input("Asal\t\t:")

call = Mahasiswa()
call.seta(a)
call.setb(b)
call.setc(c)
call.setd(d)
call.sete(e)
call.setf(f)
call.setg(g)

outa = call.geta()
outb = call.getb()
outc = call.getc()
outd = call.getd()
oute = call.gete()
outf = call.getf()
outg = call.getg()
    
print()
print(f"Data Praktikan Kelompok {outc}".center(40,"-"))
print("Anggota 1:")
print("Nama:", outa)
print("NIM:", outb)
print("Asal:", outf)
print('-' * 40)
print("Anggota 2:")
print("Nama:", outd)
print("NIM:", oute)
print("Asal:", outg)
print('-' * 40)

print()
print(f"Nama anggota pertama adalah {outa} dengan NIM {outb} dari {outf}")
print(f"Nama anggota Kedua adalah {outd} dengan NIM {oute} dari {outg}")